projects/aca-shared/rules/src/app.rules.ts

```diff
@@ -624,3 +626,19 @@ export function canOpenWithOffice(context: AcaRuleContext): boolean {
 
   return context.permissions.check(file, ['update']);
 }
+
+function isSmartFolder(context: RuleContext): boolean {
+  if (context.selection && !context.selection.isEmpty) {
```

Comment:
I think it can be shortened to `!context.selection?.isEmpty`

Reply:
thanks for a review!